spec/unit/resources/switch/update_port_spec.rb

```diff
@@ -0,0 +1,46 @@
+require_relative './../../../spec_helper'
```

Comment:
Do you think that can be good implement a shared test to be used by interconnect and switch, once those oneview chef resources are using the same Mixin to appy action `update_port`?
As the action is the same, sharing same code, maybe is not necessary duplicate the code of test, one shared test can be sufficient, the only difference seems be the parameters used in the test and this can be passed using variables `let`.
Let me know your opinion.